elastic/apm-server
# [faas] add `id` field

## Motivation/summary

`id` identifies one or a grouping of functions as
a service.

## Checklist


## How to test these changes

send some events with `faas.id` to the apm-server. verify that the indexed documents have `faas.id`.

## Related issues
